src/nginxconfig/templates/domain_sections/php.vue

```diff
@@ -207,8 +307,10 @@ THE SOFTWARE.
                         this.$props.data.joomlaRules.enabled = true;
                         this.$props.data.joomlaRules.computed = this.$props.data.joomlaRules.value;
                     } else {
-                        this.$props.data.wordPressRules.enabled = false;
-                        this.$props.data.wordPressRules.computed = false;
```

Comment:
It looks like we lost the wordpress rule disabling here.

Reply:
You really have a eagle eyes !